web-marketplace/src/pages/CreateOrganization/CreateOrganization.tsx

```diff
@@ -112,19 +117,20 @@ function CreateOrganizationContent({
           organizationId={organizationId}
           setOrganizationId={setOrganizationId}
           onTransferProfile={handleApplyTransferProfile}
-          data={data}
-          handleSaveNext={handleSaveNext}
-          onValidityChange={setIsStepValid}
         />
       )}
-      {activeStep === 1 && <MigrateProjectsStep />}
-      {activeStep === 2 && <PersonalInfoStep />}
-      {activeStep === 3 && <InviteMembersStep />}
+      {steps[activeStep].id === MIGRATE_PROJECTS_FORM_ID && (
+        <MigrateProjectsStep ref={activeRef} />
+      )}
```

Comment:
since "migrate projects" step gets hidden if there's no project, that means we can't rely only on `activeStep` value to display the right form